bindings/c/foundationdb/fdb_c.h

```diff
@@ -288,6 +288,8 @@ DLLEXPORT WARN_UNUSED_RESULT fdb_error_t fdb_result_get_keyvalue_array(FDBResult
 
 DLLEXPORT WARN_UNUSED_RESULT fdb_error_t fdb_create_database(const char* cluster_file_path, FDBDatabase** out_database);
 
+DLLEXPORT WARN_UNUSED_RESULT fdb_error_t fdb_create_database_from_connection_string(const char* connection_string, FDBDatabase** out_database);
```

Comment:
If we declare it here, I think we can remove the declaration in fdb_c_internal.h

Reply:
Done, thanks!